src/client/src/App.js

```diff
@@ -22,23 +24,25 @@ class App extends Component {
         }
     }
     onKeyPress = event => {
-         if (event.key === ' ') {
-              if (this.state.text === ' ') {
-                   this.setState({text: ''});
-              } else {
-                   this.handleSubmit();
-              }
-         }
+         // if (event.key === ' ') {
```

Comment:
Is the plan to re-add the commented out code?

Reply:
we need to talk about the userflow and UX, as mentioned below. do we want each word occupy one bubble, and as user type each word, more bubbles will pop out?

Reply:
Does this answer: https://github.com/imann24/now/pull/15/files#r315968692 ?

Reply:
Planning to remove this code?